doc/presentación/defensa.tex

```diff
@@ -0,0 +1,247 @@
+%% (ioannis.liaperdos@gmail.com)
+%%
+%% Last update: 22/06/2017 (English Support)
+%% Source: https://es.overleaf.com/latex/templates/thesis-presentation-template-beamer-english-version-dpt-of-computer-engineering-technological-educational-institute-of-peloponnese/vwhtyshhtqmg
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%
+\PassOptionsToPackage{unicode}{hyperref}
+\PassOptionsToPackage{naturalnames}{hyperref}
+\documentclass{beamer} 
+\usepackage{babel}
+\usepackage[utf8]{inputenc}
+
+
+%%% FONT SELECTION %%%%%%%%%%%%%%%%%
+%%% we choose a sans font %%%%%%%%%%
+\usepackage{kmath,kerkis} 
+%\usepackage[default]{gfsneohellenic} 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\usepackage{color}
+\usepackage{amsmath}
+\usepackage{amssymb}
+
+\usepackage{epstopdf}
+\usepackage{graphicx}
+\graphicspath{{./img/}}
+
+%%
+% load TEI-Pel - specific layout
+\usepackage{TeiPel_En_Beamer_Layout}
+\setTeipelLayout{draft,newlogo}% options: "draft", "newlogo"
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Thesis Info %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+	% title
+		\title[Análisis de redes causales en deportes de equipo]{Análisis de redes causales en deportes de equipo}	
+	% author 
+    % (In the mandatory argument "{}", separate multiple
+    % authors with "\and" - use "\\" for better author name formatting
+    % in the title page. In the optional argument "[]" include all
+	% author names, with no "\and" or text formatting macros.)
+	% Example: 
+    %\author[A. Author Albert Einstein]{Anthony Author \and Albert Einstein}
+		\author[E. Merelo]{Elena Merelo}
+	% supervisor	
+		% \supervisor{Supervisor}{Mister Supervisor}{Professor}
+	% date
+		\presentationDate{7 de septiembre de 2022}
+%%%%%%%%%%%%%%%%
+
+\begin{document}
+
+% typeset front slides
+	\typesetFrontSlides
+
+%%%%%%%%%%%%%%%%
+% Your Slides Start here:
+
+%%%%
+\section{Introducción}
+
+%%
+\subsection[Problema]{Descripción del problema}
+
+\begin{frame}{Motivación}
+	\framesubtitle{El fútbol es un deporte de equipo \textit{complicado}}
+    Como parte del cuerpo técnico de un equipo habrá que decidir:
+    \begin{itemize}
+		\item Quiénes jugarán
+		\pause
+		\item Dónde se posicionarán
+		\pause
+        \item Cuándo y qué cambios habrá
+		\pause
+        \item Alineación por la que se abogará
+        \pause
+        \item Así como a quién fichar, a quién vender, qué hacer en los entrenamientos 
+        \pause
+        \item Entran muchas variables \textit{en juego}
+	\end{itemize}
+\end{frame}
+
+\begin{frame}{Objetivos del trabajo}
+	\begin{itemize}
+		\item <1->Entender cómo se pueden aplicar técnicas estadísticas en deportes de equipo, 
+        y adaptar resultados en ese campo a los tipos de datos que existen aquí.
+		\item <2->Crear una herramienta para que personas del equipo técnico de un equipo
```

Comment:
Igual aquí. Tendrás que decir que el análisis de redes sociales es rápido y se puede hacer en tiempo real, siempre que se tengan datos en tiempo real. También se pueden hacer análisis al final de cada uno de los tiempos, o del espacio de tiempo que uno desee.

Reply:
eso lo tenía pensado comentar en trabajos futuros, is that ok?